engine/src/main/java/org/terasology/config/flexible/FlexibleConfig.java

```diff
@@ -67,7 +67,7 @@
     Map<SimpleUri, Setting> getSettings();
 
     /**
-     * Returns a potentially verbose human-readable description of the purpose of this {@link FlexibleConfig}.
+     * Returns a potentially verbose, human-readable description regarding the purpose of this {@link FlexibleConfig}.
      * This description is also written to file when the {@link FlexibleConfig} is saved.
```

Comment:
How? I haven't seen changes to the save method. Am I missing something?

Reply:
I forgot to push the last commit, sorry. I shall do that now.